variables.tf

```diff
@@ -137,3 +137,9 @@ variable "labels" {
   description = "Labels to apply to this address."
   default     = {}
 }
+
+variable "descriptions" {
+  description = "A list of descriptions to add to each address."
+  type        = list(string)
+  default     = [""]
```

Comment:
Perhaps better to have the default list empty, rather than a single zero length string.

```suggestion
  default     = []
```